src/lib/Zikula/Bundle/CoreBundle/Twig/Extension/PagerExtension.php

```diff
@@ -0,0 +1,399 @@
+<?php
```

Comment:
I think the pager should be re-done / have more comments, etc. I remember problems with 0 vs. 1 as starting position.